tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -201,6 +201,19 @@ public function adminDeletesGroupUsingTheGraphApi(
 		}
 	}
 
+	/**
+	 *
+	 * @param string $userUUIDv4
+	 *
+	 * @return int
+	 * @throws Exception
+	 * @throws GuzzleException
+	 */
+	public function checkUUIDv4PatternForUserId(string $userUUIDv4): int {
```

Comment:
```suggestion
	public function checkUUIDv4PatternForUserId(string $user_id) int {
```

Reply:
Please keep on using camel case `$user_id` will break the system